google/ggrc-core
# QUICK-FIX Remove obsolete import cache

# Issue description

Since we have removed object owners, the owners cache is never used
anymore and is just dead code.

# Steps to test the changes

Check that the part of the code is really never used. I have no idea what to check in the app

# Solution description

Removed dead code.

# Sanity checklist

- [x] I have clicked through the app to make sure my changes work and not break the app. - tried import/export but I don't know what could be touched by this code.



# PR Review checklist
